### DISCUSSION TITLE
Work like a charm, but what about battery life?

### DISCUSSION THREAD

**Discussion by mmornati** [brianschrameck/arlo-cam-api#9] [General]
Thanks a lot for this work. 
I can share I was able to let it work with **Arlo Pro 3** devices which are linked to a **VMB4540r2** and it works like a charm.

But my concern is than related to the battery drain of the arlo camera. As the application is no more available, I installed Frigate and the camera flow was OK, events detected and session recorded... but, no idea why, even without any events or record, the battery was almost empty at the end of the first day (from 100%). 
I'm maybe missing anything somewhere or misconfiguring the devices... what about your configuration? What are you using? What is the battery lifetime?

Thanks!

**Comment by brianschrameck** [comment]
Last time I checked, Frigate is designed for all cameras to be streaming 24/7, and Frigate uses its own motion and object detection mechanisms. You can disable motion and object detection and create your own events via the Frigate API, but it still keeps the streams running.

I could be wrong about this, but given the behavior you are seeing that's my hunch.

Even without Frigate, battery life is much worse than Arlo's native implementation. The cameras go into some sort of ultra low-power standby state when using the Arlo base station, but I haven't been able to replicate that using an emulated base station API. When I was running on battery I was only getting a couple of weeks, maybe less. I can't really recall.

You'll want to work out a way to power the cameras, for sure.
